docker_rsync.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+from os.path import abspath, join
+import docker
+
+from logger import log_from_docker
+
+client = docker.from_env()
+
+
+def run_rsync(volumes, from_path, to_path, relative):
+    # rsync options:
+    # -v = verbose - give info about what files are being transferred and a brief summary at the end
+    # -r = copy directories recursively
+    # -e = specify remote shell program explicitly (i.e. ssh as opposed to the default rsh)
+    # --delete = delete destination files not in source
+    cmd = ["rsync", "-rv", "-e", "ssh", from_path, to_path, "--delete"]
+
+    if relative:
+        cmd = ["rsync", "-rv", "-e", "ssh", "--relative", from_path, to_path, "--delete"]
+
+    container = client.containers.run("instrumentisto/rsync-ssh", command=cmd, volumes=volumes,
+                                      detach=True)
+
+    log_from_docker(container)
+
+
+def create_volumes(settings, local_volume, volume_mode):
```

Comment:
This is a misleading function name - it doesn't create volumes, it returns the args that the docker API takes to create them. Plus, presumably the volumes are not always created? After the first time they are just being mapped into the container.

So maybe `get_volumes_args`?